parser.c

```diff
@@ -68,7 +68,7 @@ int	parse_term(char **str, t_tree **left_tree)
 		tok = scan_token(*str);
 		if (!tok)
 			return (0);
-		if (!(tok->type == operation && (tok->value.c == '*' || tok->value.c == '/')))
+		if (!(tok->type == operation && (tok->value.c == '*' || tok->value.c == '/' || tok->value.c == '^')))
```

Comment:
The priorities of operations aren't respected. In the case `./eval "4 * 3 ^ 2"`, 3 ^2 should have priority over multiplication. I think power operation cannot be handled in the parse_term function.

Reply:
yes, I think it's easier to remove the operator in this pull request and sort it out later.